code/controllers/subsystem/persistence.dm

```diff
@@ -294,6 +295,42 @@ SUBSYSTEM_DEF(persistence)
 	for(var/obj/structure/sign/painting/P in painting_frames)
 		P.load_persistent()
 
+	json_file = file("data/pending_art_metacoin.json")
+	if(fexists(json_file))
+		pending_art_metacoin = json_decode(file2text(json_file))
+
+	if(paintings && paintings["library"] && paintings["library"][1] && !paintings["library"][1]["price"]) //this should only ever happen once - adding owner and price values to legacy paintings
+		message_admins("Legacy paintings detected, adding owner and price values")
+		log_game("Legacy paintings detected, adding owner and price values")
+		for(var/entry in paintings["library"])
+			entry["owner"] = entry["author"]
+			entry["price"] = 500
```

Comment:
Hm, this should probably have some safety against those 2 entries not existing when its actually used.